src/main/java/in/twizmwaz/cardinal/event/flag/FlagEvent.java

```diff
@@ -0,0 +1,22 @@
+package in.twizmwaz.cardinal.event.flag;
+
+import in.twizmwaz.cardinal.module.modules.ctf.FlagObjective;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+public abstract class FlagEvent extends Event {
```

Comment:
This is an object, not an interface, so we can include a reference to flag here.